### PR TITLE
[9.0][ADD] mrp_disable_force_availability_button

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
MRP Disable Force Availability Button
===============================

This module creates a security group that limits the visibility of "Force
availability" button on manufacture order to those that belong to
this group.

### CODE REVIEW

File: mrp_disable_force_availability_button/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/129/9.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/manufacturing/issues>`_. In case of trouble, please
```
Comment (MiquelRForgeFlow) [inline mrp_disable_force_availability_button/README.rst]: manufacturing → manufacture

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:  Code review. Really a bad practice to force availability.

File: mrp_disable_force_availability_button/README.rst (hunk 1 above)
Comment (LoisRForgeFlow) [reply mrp_disable_force_availability_button/README.rst]: fixed.

Thanks!

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**